Source/FieldSolver/SpectralSolver/SpectralHankelTransform/HankelTransform.cpp

```diff
@@ -198,38 +198,14 @@ HankelTransform::HankelForwardTransform (amrex::FArrayBox const& F, int const F_
     AMREX_ALWAYS_ASSERT(ngr >= 0);
     AMREX_ALWAYS_ASSERT(F_box.bigEnd(0)+1 >= m_nr);
 
-#ifndef AMREX_USE_GPU
-    // On CPU, the blas::gemm is significantly faster
-
     // Note that M is flagged to be transposed since it has dimensions (m_nr, m_nk)
+    amrex::Gpu::synchronize();
     blas::gemm(blas::Layout::ColMajor, blas::Op::Trans, blas::Op::NoTrans,
```

Comment:
let's try to create a `blas::Queue( int device_id, int64_t batch_size=0 )` object here

Reply:
done :)